src/parsers/forecast_parser.jl

```diff
@@ -22,44 +66,115 @@ function ForecastInfos()
 end
 
 """
-    forecast_csv_parser!(sys::System,
-                         directory_or_file::AbstractString,
-                         simulation="Simulation",
-                         category::Type{<:Component}=Component,
-                         label="scalingfactor";
-                         resolution=nothing,
-                         kwargs...)
+    add_forecasts!(sys::System, metadata_file::AbstractString; resolution=nothing)
```

Comment:
We should clarify a bit more the expected use case here is when the user wants to parse directly from files

Reply:
Let's have a team discussion on whether we create many methods on the same function or create more unique function names.